src/main/java/com/commercetools/helper/mapper/impl/payment/DefaultPaymentMapperImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.commercetools.helper.mapper.impl.payment;
+
+import com.commercetools.helper.formatter.PaypalPlusFormatter;
+import com.commercetools.helper.mapper.AddressMapper;
+import com.commercetools.helper.mapper.PaymentMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.Nonnull;
+
+import static com.commercetools.payment.constants.CtpToPaypalPlusPaymentMethodsMapping.DEFAULT;
+
+/**
+ * Default CTP to PP payment mapper.
+ */
+@Component
+public class DefaultPaymentMapperImpl extends BasePaymentMapperImpl implements PaymentMapper {
```

Comment:
Actually it's a Paypal Plus mapper, right?

Reply:
what do you understand under _Paypal Plus mapper_?

Reply:
Maps an object to Paypal Plus format.

Reply:
Than yes - `DefaultPaymentMapperImpl` corresponds on what you understand as _Paypal Plus mapper_, this is explicitly mentioned on the comment above:
>Default CTP to PP payment mapper